src/diff.js

```diff
@@ -29,6 +29,10 @@
  *
  * QUnit.diff( "the quick brown fox jumped over", "the quick fox jumps over" ) === "the  quick <del>brown </del> fox jump<ins>s</ins><del>ed</del over"
  */
+
+// Don't load QUnit.diff on non-Browser environments
+if ( typeof window !== "undefined" && window.document ) {
```

Comment:
This seems strange. Just because `QUnit.diff` currently outputs HTML doesn't mean we can't use it for CLI reporters in the future. Even ignoring that, what's the issue of running the code in node?